src/index.js

```diff
@@ -1,18 +1,51 @@
-import _ from 'lodash';
-import printMe from './print.js';
+import './style.css';
 
-function component() {
-  const element = document.createElement('div');
-  const btn = document.createElement('button');
+const tasks = [
+  {
+    id: 0,
+    description: 'Do laundry',
+    completed: false,
+  },
+  {
+    id: 1,
+    description: 'Buy groceries',
+    completed: false,
+  },
+  {
+    id: 2,
+    description: 'Go for a run',
+    completed: true,
+  },
+];
 
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+class Task {
+  constructor(description) {
+    this.id = tasks.length;
+    this.description = description;
+    this.completed = false;
+  }
 
-  btn.innerHTML = 'Click me and check the console!';
-  btn.onclick = printMe;
-
-  element.appendChild(btn);
-
-  return element;
+  static toggleTaskStatus(id) {
+    tasks.forEach((task) => {
+      if (task.id === id) {
+        task.completed = !task.completed;
+      }
+    });
+  }
 }
 
-document.body.appendChild(component());
\ No newline at end of file
+window.onload = () => {
+  const todoList = document.querySelector('.todo-list');
+  Task.tasks.forEach((task) => {
+    const li = document.createElement('li');
+    li.className = 'todo-item';
+    li.innerHTML = ` 
+      <label data-id=${task.id} class="${task.completed ? 'todo-completed' : ''}"> 
+      <input type="checkbox" class="todo-item-check" ${task.completed ? 'checked' : ''}> 
+      ${task.description}    
+      </label>
+      <i class="fas fa-ellipsis-v item-edit-icon"></i> 
+      `;
+    todoList.appendChild(li);
+  });
+};
```

Comment:
- [ ] Kindly make sure that On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the index values for each task. but in the case of yours when I run the project it throws an error as shown below and kindly make sure that the task array are rendered dynamically to the DOM.Please Fix the issues.
![Capture](https://user-images.githubusercontent.com/93598189/235818396-a3c02736-41c5-4679-bf7f-d03909f89836.PNG)